contributions/catalog/controllers/contribute.py

```diff
@@ -62,8 +62,8 @@ def create():
 
         # add contributionAdmins to the json_contiubtion
         contribution = jsonutil.add_contribution_admins(contribution)
-
         json_contribution = json.dumps(contribution, indent=4)
+        print(json_contribution)
```

Comment:
Please remove print statements. You may use logging instead. Thanks.

Reply:
removed.